test/features/deploy_quarkus_service.feature

```diff
@@ -154,3 +154,47 @@ Feature: Deploy quarkus service
     Examples:
       | native  | minutes |
       | enabled | 20      |
+
+  @usertasks
+  Scenario Outline: Deploy process-quarkus-example service to complete user tasks and native <native>
```

Comment:
@radtriste @Sgitario Does it have sense to test this functionality in Operator Cucumber tests?
IMHO it would have sense to check integration with Keycloak related to user tasks and authentication/authorization. Checking generic tser task functionality should be covered in Kogito runtime.

Reply:
I think we may keep them disabled with comment to adjust and enable them once they are enhanced with Keycloak integration.

Reply:
I'm a bit out of context here. What you mean is that the current "user" parameter will be replaced by getting this user from Keycloak, am I right?

If so, I think it makes sense to keep these tests and once we have Keycloak, we can just update these tests and this could help in this work.

Reply:
No, the current user will stay like it is now.
The scenario will be enhanced by deploying Keycloak, specifying user `john` there his credentials and possibly roles. 
Then in the scenario you will request orders as user `john` (supplying his credentials?) and making sure that you can see just orders which are assigned to him.